### ISSUE TITLE
Positional selectors are not detected when using concatenation to build the selector

### ISSUE THREAD

**Issue by stof** [wikimedia/eslint-plugin-no-jquery#101]
```js
this.$stickyCol
  .find('thead th:gt(' + colCount + ')').remove()
```

this does not report the usage of `:gt()` in the `.find()` call.
